pkgs/by-name/tt/tts-mod-vault/package.nix

```
@@ -0,0 +1,47 @@
+{
+  lib,
+  flutter329,
+  makeDesktopItem,
+  copyDesktopItems,
+  fetchFromGitHub,
+}:
+
+flutter329.buildFlutterApplication rec {
+  pname = "tts-mod-vault";
+  version = "1.3.0";
+
+  src = fetchFromGitHub {
+    owner = "markomijic";
+    repo = "TTS-Mod-Vault";
+    tag = "v${version}";
+    hash = "sha256-BTs+4QeyVJeg415uiNXww8twQwUInHfB8voWJjeVs20=";
+  };
+
+  pubspecLock = lib.importJSON ./pubspec.lock.json;
+
+  desktopItems = [
+    (makeDesktopItem {
+      name = "tts-mod-vault";
+      exec = "tts-mod-vault";
+      icon = "tts-mod-vault";
+      comment = "Tabletop Simulator Mod Vault";
+      desktopName = "TTS Mod Vault";
+      categories = [ "Utility" ];
+    })
+  ];
+
+  nativeBuildInputs = [ copyDesktopItems ];
+
+  postInstall = ''
+    install -m 444 -D assets/icon/tts_mod_vault_icon.png $out/share/icons/hicolor/512x512/apps/tts-mod-vault.png
```

this icon is 1024x1024, not 512x512
```suggestion
    install -m 444 -D assets/icon/tts_mod_vault_icon.png $out/share/icons/hicolor/1024x1024/apps/tts-mod-vault.png
```